tests/plugins/test_precision_plugin.py

```diff
@@ -0,0 +1,26 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from inspect import signature
+
+from pytorch_lightning.plugins.precision import PrecisionPlugin
+
+
+def test_precision_clip_gradients_signature():
+
+    expected_params_list = ['self', 'model', 'optimizer', 'clip_val', 'norm_type']
+
+    params = signature(PrecisionPlugin.clip_gradients).parameters
+    params_list = [param.name for param in params.values()]
+
+    assert params_list == expected_params_list
```

Comment:
This test seems pointless to me. Why do we test the function signature?

The premise of testing is to compare the actual outputs to expected outputs given some inputs.

But we have neither inputs nor outputs here

Reply:
The idea was that, without the changes in the PR, the test should fail.
Also, we already have tests to test the working of clip gradients & Precision Plugins.
The test is for this specific change. But yes, the redundancy of it is debatable.